### DISCUSSION TITLE
Command text color, fish update caused my text colors to vanish

### DISCUSSION THREAD

**Discussion by Lilja** [fish-shell/fish-shell#9353] [Q&A] [answered]
I updated fish from 3.1.0 something to 3.5.2. Doing so changed my colors:
![firefox_2022-11-15_08-33-02](https://user-images.githubusercontent.com/6134511/201861078-37902e8a-b792-4434-bbb1-96da0fbda9c9.png)

This is how I normally my command text:
![image](https://user-images.githubusercontent.com/6134511/201861795-39f631e3-2180-47c2-bf60-9d0266ec5f14.png)

What am I missing? Is this set from the terminal emulator or from fish?

**Comment by faho** [comment, marked as answer]
>3.5.2

3.5.2 doesn't exist, you mean 3.5.1.

>Is this set from the terminal emulator or from fish?

In this case, most likely the terminal. Fish tells `ls` to turn on "color" by running `ls --color=auto`, but the normal permissions text and such would usually be the "normal" color, which is configured in your terminal.

That means the fish upgrade isn't the cause of the change, you probably updated your terminal at the same time.
